src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/StringLiteralEqualityCheck.java

```diff
@@ -46,7 +51,18 @@
  * <pre>
  * &lt;module name=&quot;StringLiteralEquality&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of violations:
+ * </p>
+ * <pre>
+ * if (status == "done") {} // violation
  *
+ * boolean flag = (status == "done") // violation
```

Comment:
Add 1 more example of `!=` that will cause a violation. Something distinct from the others like a loop.